langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -47,14 +59,26 @@ public isolated function iterator(string str) returns object {
 
 # Concatenates zero or more strings.
 #
+# ```ballerina
+# string hostname = "http://worldtimeapi.org";
+# string continent = "Asia";
+# string city = "Colombo";
+# string worldTimeAPIUrl = hostname.concat("/api/timezone/", continent, "/", city);
+#
+# ```
 # + strs - strings to be concatenated
 # + return - concatenation of all of the parameter `strs`; empty string if parameter `strs` is empty
 public isolated function concat(string... strs) returns string = @java:Method {
     'class: "org.ballerinalang.langlib.string.Concat",
     name: "concat"
 } external;
 
-# Returns the code point of a character in a string.
+# Returns the unicode code point of a character in a string.
```

Comment:
Shall we make first letter of `unicode` to uppercase in here.
We need to be consistent with L85 i think.

Reply:
Fixed.